src/frontend/pages/Movies.js

```diff
@@ -0,0 +1,181 @@
+import React, { useEffect, useState } from 'react';
+import { Layout, Input, Row, Col, Card, Tag, Spin, Modal, Typography, Button } from 'antd';
+import { Link } from "react-router-dom";
+import 'antd/dist/antd.css';
+
+const { Content } = Layout;
+const { Search } = Input;
+const { Meta } = Card;
+const TextTitle = Typography.Title;
+
+
+const SearchBox = ({searchHandler}) => {
+    return (
+        <Row>
+            <Col span={12} offset={6}>
+                <Search
+                    placeholder="Search for movies to schedule!"
+                    enterButton="Search"
+                    size="large"
+                    onSearch={value => searchHandler(value)}
+                />
+            </Col>
+        </Row>
+    )
+}
+
+const MovieCard = ({Title, imdbID, Poster, ShowDetails, DetailRequest, ActivateModal}) => {
+
+    const clickHandler = () => {
+        
+        ActivateModal(true);
+        DetailRequest(true);
+
+        fetch(`http://www.omdbapi.com/?i=${imdbID}&apikey=cde43fc8`)
+        .then(resp => resp)
+        .then(resp => resp.json())
+        .then(response => {
+            DetailRequest(false);
+            ShowDetails(response);
+        })
+    }
+
+    return (
+        <Col style={{margin: '50px'}} span={3}>
+            <div>
+                <Card
+                    style={{ width: 300 }}
+                    cover={
+                        <img
+                            alt={Title}
+                            src={Poster === 'N/A' ? 'https://placehold.it/198x264&text=Image+Not+Found' : Poster}
+                        />
+                    }
+                    onClick={() => clickHandler()}
+                >
+                    <Meta
+                        title={Title}
+                    />
+                </Card>
+            </div>
+        </Col>
+    )
+}
+
+const MovieDetail = ({Title, Actors, Released, Rated, Runtime, Genre, Poster, Plot, }) => {

```

Comment:
We probably want this to be a separate component so we can reuse it on other pages.